ravenpy/models/emulators/sacsma.py

```diff
@@ -0,0 +1,672 @@
+        #                                            lakes and vegetation that can deplete stream flow by evapotranspiration.; percent/100;
+        #                                            default=0.0
+        # para_x16 0.0    0.4      # RSERV;          Fraction of lower zone free water unavailable for transpiration; percent/100; default=0.3
+        # n/a      0.0    1.0      # UH1;            The first  component of the unit hydrograph, i.e. the proportion of instantaneous runoff
+        #                                            not lagged; percent/100; default=1.0
+        # n/a      0.0    1.0      # UH2;            The second component of the unit hydrograph, i.e. the proportion of instantaneous runoff
+        #                                            not lagged; percent/100; default=1.0
+        # n/a      0.0    1.0      # UH3;            The third  component of the unit hydrograph, i.e. the proportion of instantaneous runoff
+        #                                            not lagged; percent/100; default=1.0
+        # n/a      0.0    1.0      # UH4;            The fourth component of the unit hydrograph, i.e. the proportion of instantaneous runoff
+        #                                            not lagged; percent/100; default=1.0
+        # n/a      0.0    1.0      # UH5;            The fifth  component of the unit hydrograph, i.e. the proportion of instantaneous runoff
+        #                                            not lagged; percent/100; default=1.0
+        # para_x17 0.0    8.0      # MELT_FACTOR;    maximum snow melt factor used in degree day models (not in original SAC-SMA
+        #                                            model); mm/d/C; default=1.5
+        # para_x18 0.3    20.0     # GAMMA_SHAPE;    used to build unit hydrograph, LandUseParameterList; none ; default=1.0
+        # para_x19 0.01   5.0      # GAMMA_SCALE;    used to build unit hydrograph, LandUseParameterList; none ; default=1.0
+        # para_x20 0.8    1.2      # RAINCORRECTION; Muliplier to correct rain, Gauge; none ; default=1.0
+        # para_x21 0.8    1.2      # SNOWCORRECTION; Muliplier to correct snow, Gauge; none ; default=1.0
+
+        # tied parameters:
+        # (it is important for OSTRICH to find every parameter place holder somewhere in this file)
+        # (without this parameters that are used to derive parameters wouldn't be detectable by Ostrich)
+        #    para_x04_mm       = par_soil0_mm     = para_x04 * 1000       =  par_x04 * 1000
+        #    para_x06_mm       = par_soil0_mm     = para_x06 * 1000       =  par_x06 * 1000
+        #    para_bf_loss_frac = PAR_BF_LOSS_FRAC = para_x12/(1+para_x12) = par_x12/(1+par_x12)
+        #    para_pow_x01      = POW_X01          = 10^(para_x01)         = 10^par_x01
+        #    para_pow_x02      = POW_X02          = 10^(para_x02)         = 10^par_x02
+        #    para_pow_x03      = POW_X03          = 10^(para_x03)         = 10^par_x03
+
```

Comment:
above 4 parameters get also not filled in properly. Two out of the four values need to be templated using something like `{}`